var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -76,10 +76,11 @@ class Openmpi(AutotoolsPackage):
     """
 
     homepage = "http://www.open-mpi.org"
-    url = "https://www.open-mpi.org/software/ompi/v2.1/downloads/openmpi-2.1.1.tar.bz2"
+    url = "https://www.open-mpi.org/software/ompi/v3.0/downloads/openmpi-3.0.0.tar.bz2"
```

Comment:
Out of curiosity did the install not work until you changed this? I'm asking because you shouldn't have to alter this URL (Spack should automatically substitute the version into it).

Reply:
No I did not have problems.  I just thought it should be changed to reflect the latest and
"greatest" release.  If you'd rather leave it the same I can revert that part.